src/app/domains/categories/domains.transaction-categories.mocks.ts

```diff
@@ -0,0 +1,13 @@
+import { TransactionCategory } from 'src/app/pages/categories/transaction-category.model';
+import { WalletTransactionType } from '../transactions/domains.transactions.constants';
+import { ITransactionCategoryApiResponse } from './domains.transaction-categories.types';
+
+export const transactionCategoriesMock: ITransactionCategoryApiResponse[] = [
+  { id: 1, name: 'Nic', type: WalletTransactionType.Income },
+  { id: 2, name: 'aNic2', type: WalletTransactionType.Income },
+  { id: 3, name: 'bNic3', type: WalletTransactionType.Expense },
+];
+
+export const transactionCategoriesObjectsMock: TransactionCategory[] = [
+  ...transactionCategoriesMock.map( (c: ITransactionCategoryApiResponse) => new TransactionCategory(c.id, c.name, c.type) ),
```

Comment:
`map` is already creating new array, you don't have to use `...` and place all of it inside `[]`